packs/data/feats.db/brightsoul.json

```diff
@@ -19,12 +19,36 @@
         "level": {
             "value": 1
         },
-        "location": "",
+        "location": null,
         "onlyLevel1": true,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
```

Comment:
You should add a GrantItem for the Light Spell Effect or a TokenLight that does the same thing.